spring-rabbit/src/main/java/org/springframework/amqp/rabbit/log4j2/AmqpAppender.java

```diff
@@ -484,6 +547,58 @@ private void activateOptions() {
 			this.senderPool = Executors.newCachedThreadPool();
 		}
 
+		/**
+		 * Create the {@link ConnectionFactory}.
+		 *
+		 * @return a {@link ConnectionFactory}.
+		 */
+		protected ConnectionFactory createRabbitConnectionFactory() {
+			RabbitConnectionFactoryBean factoryBean = new RabbitConnectionFactoryBean();
+			configureRabbitConnectionFactory(factoryBean);
+			try {
+				factoryBean.afterPropertiesSet();
+				return factoryBean.getObject();
+			}
+			catch (Exception e) {
+				LogFactory.getLog("log4j2AppenderErrors").error("Failed to create customized Rabbit ConnectionFactory.",
```

Comment:
Not sure where I got that from; it looks like we should be using the `StatusAppender` - available as `LOGGER` in `AbstractLifecycle` (a super class of the appender) - I just took a look at the JmsAppender and it uses LOGGER.